app/assets/stylesheets/donor.css.scss

```diff
@@ -211,17 +215,17 @@
 }
 
 .donation-modify {
-  float: right;
   text-align: right;
 
   .donation-status {
-    float: right;
     font-weight: 400;
     text-align: right;
+    .canceled {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3